### PR TITLE
Add pdc

### PR DESCRIPTION
**Pull request by ingkebil**
Add support to create an encryptic archive and upload it to PDC.

### CODE REVIEW

**Comment by ingkebil** [comment]
First part done: create encrypted archive.

File: housekeeper/archive/core.py [hunk 1]
```diff
@@ -35,5 +38,6 @@ def get_archiveassets(run_id):
     for archive_type in ("data", "result"):
         category = "archive-{}".format(archive_type)
         query = api.assets(run_id=run_id, category=category)
-        asset = query.first()
-        yield archive_type, asset
+        assets = query.all()
+        for asset in assets:
+            yield archive_type, asset
```
Comment (ingkebil) [inline housekeeper/archive/core.py]: I'm not sure changing this from returning one to everything is a good idea tho. I need it for now, as to return both compilation and compilation key. But what when we have sent a compilation multiple times to PDC? What do you think?

Comment (robinandeer) [inline housekeeper/archive/core.py]: not sure I follow - what does the query return? Isn't it just both tar-archives for the run?

Comment (ingkebil) [inline housekeeper/archive/core.py]: When encrypting something, you get two files: the encrypted tar archive and the gpg key. This query will return both of those as I tag them both with `archive-{data,result}`.
So I'm unsure if `archive-{}` should only have one result.

**Pull request merged**